extract_project_descriptions_and_public_interests.py

```diff
@@ -0,0 +1,75 @@
+""" This file is for getting the project descriptions and 
+public interests from xml and save them to a csv file. """
+import time
+import re
+import requests
+from bs4 import BeautifulSoup
```

Comment:
It is a good practice to separate native imports from installed imports by empty line